src/Microsoft.AspNetCore.Server.HttpSys/HttpSysOptions.cs

```diff
@@ -55,6 +57,28 @@ public HttpSysOptions()
         public bool ThrowWriteExceptions { get; set; }
 
         /// <summary>
+        /// Gets or Sets the maximum number of concurrent connections to accept, -1 for infinate, or null for unspecified.
```

Comment:
What is the effective difference between unspecified and infinite? In Kestrel, we use null to mean infinite.

typos: "Gets or sets", "infinite".